src/domain/challenge/hub/hub.resolver.fields.ts

```diff
@@ -37,14 +41,19 @@ export class HubResolverFields {
     private groupService: UserGroupService,
     private applicationService: ApplicationService,
     private preferenceSetService: PreferenceSetService,
-    private hubService: HubService
+    private hubService: HubService,
+    private baseChallengeService: BaseChallengeService,
+    @InjectRepository(Hub)
+    private hubRepository: Repository<Hub>
```

Comment:
this is wrong, you shouldn't access repositories from here

Reply:
Moved to hub.service.ts